### ISSUE TITLE
After applying Desert high contrast theme 'View' link under logs column and 'View' button under details column looks like normal text: A11y_.NetAspireDashboard_Resource_Grid_Usability

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3360]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Tags:
#A11yTCS; #A11yUsable; #A11ySev4; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #HighContrast;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/
2. TAB to "Resources" tab item and select the tab by pressing Enter key.
3. Apply high contrast desert theme and observe the issue.

### Actual Results:
After applying Desert high contrast theme 'View' link under logs column and 'View' button under details column looks like normal text

**Similar issue is also observed in throughout the application for 'View' link and 'View' button.**

### Expected Results:
Buttons and links should be recognizable after applying high contrast desert theme.

### User Impact:
Color blind users will not be able to understand whether these are the button and links or normal texts.

### Attachment:
![3360_A11y_ NetAspireDashboard_Resource_Grid_Usability (2)](https://github.com/dotnet/aspire/assets/90901519/d3c3e1d8-17a7-4157-854b-bae4ef274946)
![3360_A11y_ NetAspireDashboard_Resource_Grid_Usability](https://github.com/dotnet/aspire/assets/90901519/e7252824-f638-41d2-8795-c62264fb3012)

**Comment by drewnoakes** [comment]
These columns no longer exist. We have a new "Actions" column with a popup menu. Closing this.
